ar_exercise/exercise.rb

```diff
@@ -26,23 +26,86 @@ class Apartment < ActiveRecord::Base
 ################################################
 # FINDING / SELECTING
 ################################################
-
+# binding.pry
 # Hint, the following methods will help: `where`, `all`, `find`, `find_by`
 
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
+puts "*" * 50
+
 all_tenants = Tenant.all
+all_apartments = Apartment.all
+all_tenants.map {|tenant| puts tenant.name + "  " + tenant.age}
+
+puts "*" * 50
 
 # get the first tenant in the DB
+first_tenant = Tenant.find(1)
```

Comment:
This assumes that the first tenant has an id of 1, however if this tenant was destroyed the first tenant might be 2 or, any other number. A better way of finding the first tenant is to do `Tenant.first`